cypress/support/commands/ui_commands/cart_commands.js

```diff
@@ -0,0 +1,9 @@
+const CART_PAGE = require('../../page_elements/cart_page').CART_PAGE;
+
+
+Cypress.Commands.add('validarImagemCart', () => {
```

Comment:
Esta command pode ser depreciada e utilizado a `validarBodyProd`.

Quando for validar uma imagem, busque a propriedade e valide o valor dela.

Ela pode existir, mas o valor pode estar errado.
[Itens diferentes no carrinho](https://prnt.sc/QOafBFNIPj5f)

Lembre-se de que a command deve ser genérica, mas a validação precisa ser específica pra não gerar falso positivo.